lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -242,7 +242,7 @@
   memory_swappiness:
     description:
         - Tune a container's memory swappiness behavior. Accepts an integer between 0 and 100.
-    default: 0
+        - If not set, the value will be inherited from the host machine.
```

Comment:
While this is true when `docker_container` creates a new container, it is not true in case it doesn't. In the latter case, the container will keep its current value (whatever it is). I've been thinking a bit on how to formulate it; I'm currently leaning towards simply not mentioning the case when it's not specified (i.e. leave this new sentence away, but also drop the `default: 0`).

Reply:
Right. How about?

```
- If not set, the value will be remain the same if container exists and will be inherited from the host machine if it doesn't
```